src/Pages/Shared/ConfigMapSecret/utils.ts

```diff
@@ -488,22 +492,91 @@ export const getConfigMapSecretPayload = ({
     return payload
 }
 
+const getConfigMapSecretDecodedData = <IsDraft extends boolean = false>({
+    configMapSecretData,
+    isDraft,
+    isSecret,
+}: ConfigMapSecretDecodedDataProps<IsDraft>): ConfigMapSecretDecodedDataReturnType<IsDraft> => {
+    if (!configMapSecretData || !isSecret || configMapSecretData.unAuthorized) {
+        return configMapSecretData
```

Comment:
it will return null or undefined in case of no configMapSecretData. Is it expected?